README.rst

```diff
@@ -64,42 +73,43 @@ Additional optional arguments
- skip_row_id                   Skip the row_id column in output.
- output_delimiter=<delimiter>  Specifies delimiter for output CSV. The special keyword "tab" can be used to indicate a tab delimited csv.
- timeout=<timeout>             The time, in seconds, that DataRobot tries to make a connection to satisfy a prediction request. When the timeout expires, the client (the batch_scoring command) closes the connection and retries, up to the number of times defined by the value of ``<n_retry>``. The default value is 30 seconds.
- delimiter=<delimiter>         Specifies the delimiter to recognize in the input .csv file (e.g., "--delimiter=,"). If not specified, the script tries to automatically determine the delimiter. The special keyword "tab" can be used to indicate a tab-delimited csv.
- resume                        Starts the prediction from the point at which it was halted. If the prediction stopped, for example due to error or network connection issue, you can run the same command with all the same arguments plus this ``resume`` argument. If you do not include this argument, and the script detects a previous script was interrupted mid-execution, DataRobot prompts whether to resume. When resuming a script, you cannot change the ``dataset_filepath``,  ``model_id``, ``project_id``, ``n_samples``, or ``keep_cols``.
- help                          Show usage help for the command.
- fast                          *Experimental*: Uses a faster csv processor. Note that this method does not support multiline csv.
- stdout                        Send all log messages to stdout.
- auto_sample                   Override the ``<n_samples>`` value and instead use chunks of roughly 1.5 MB to improve throughput. On by default.
- encoding                      Declare the dataset encoding. If an encoding is not provided, the batch_scoring script attempts to detect it (e.g., "utf-8", "latin-1", or "iso2022_jp"). `See the Python docs for a list of valid encodings <https://docs.python.org/3/library/codecs.html#standard-encodings>`_.
- skip_dialect                  Tell the batch_scoring script to skip csv dialect detection.
-============================== ===========
+============================== ========== ========= ===========
+  Argument                     Standalone Dedicated Description
+------------------------------ ---------- --------- -----------
+ host=<host>                      \+         \+     Specifies the hostname of the prediction API endpoint (the location of the data from where to get predictions).
+ user=<user>                      \-         \+     Specifies the username used to acquire the api-token. Use quotes if the name contains spaces.
+ <import_id>                      \+         \-     Specifies imported model unique identification string. You can ask for ID your prediction administrator who did the import procedure.
```

Comment:
Specifies the unique ID for the imported model. If unknown, ask your prediction administrator (the person responsible for the import procedure).